src/NServiceBus.Core/Licensing/ConfigureLicenseExtensions.cs

```diff
@@ -30,9 +27,7 @@ public static void License(this EndpointConfiguration config, string licenseText
     /// <remarks>The license provided via code-first API takes precedence over other license sources.</remarks>
     public static void LicensePath(this EndpointConfiguration config, string licenseFile)
     {
-        ArgumentNullException.ThrowIfNull(config);
-        ArgumentException.ThrowIfNullOrWhiteSpace(licenseFile);
-
+        // Intentionally not doing validation as passed value could be dynamic and (temporarily) be invalid
```

Comment:
What is strange here is that currently we do not do a `File.Exist`. If we defer that we could also defer ANY validation. However, now including a `File.Exist` is a breaking change.

Reply:
It is a tricky situation, it may be nice to involve others to see their thoughts